libs/ml/include/ml/utilities/graph_builder.hpp

```diff
@@ -18,65 +18,9 @@
 //------------------------------------------------------------------------------
 
 #include "ml/core/graph.hpp"
-#include "ml/ops/metrics/categorical_accuracy.hpp"
-
-#include "ml/layers/PRelu.hpp"
-#include "ml/layers/convolution_1d.hpp"
-#include "ml/layers/convolution_2d.hpp"
-#include "ml/layers/fully_connected.hpp"
-#include "ml/layers/multihead_attention.hpp"
-#include "ml/layers/scaled_dot_product_attention.hpp"
-#include "ml/layers/self_attention_encoder.hpp"
-#include "ml/layers/skip_gram.hpp"
-
+#include "ml/core/subgraph.hpp"
 #include "ml/meta/ml_type_traits.hpp"
```

Comment:
I think if you have #include "ml/core/subgraph.hpp" you do not need to include "ml/core/graph.hpp" also